scripts/docker-testnet/start.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -e
+
+export DOCKERTESTNETDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+MULTIVERSXTESTNETSCRIPTSDIR="$(dirname "$DOCKERTESTNETDIR")/testnet"
+
+source "$DOCKERTESTNETDIR/variables.sh"
+source "$DOCKERTESTNETDIR/helpers.sh"
+source "$MULTIVERSXTESTNETSCRIPTSDIR/include/config.sh"
+source "$MULTIVERSXTESTNETSCRIPTSDIR/include/build.sh"
+
+prepareFolders
+
+buildConfigGenerator
+
+generateConfig
+
+copyConfig
+
+copySeednodeConfig
```

Comment:
interesting, so you actually use a lot of the existing testnet scripts configuration functions :thinking:

Reply:
yep, it's mostly the same configuration. The only difference in configuration are the ports and the ip addresses.

Reply:
right, I've noticed that: the first question was: how was possible to function if all nodes run on the same ports. But then I've saw the IPs.